src/tools/miri/tests/native-lib/pass/ptr_write_access.rs

```diff
@@ -0,0 +1,196 @@
+// Only works on Unix targets
+//@ignore-target: windows wasm
+//@only-on-host
+
+#![feature(box_as_ptr)]
+
+use std::mem::MaybeUninit;
+use std::ptr::null;
+
+fn main() {
+    test_increment_int();
+
+    test_init_int();
+
+    test_init_array();
+
+    test_init_static_inner();
+
+    test_expose_int();
+
+    test_swap_ptr();
+
+    test_swap_nested_ptr();
+
+    test_swap_tuple();
+
+    test_overwrite_dangling();
+
+    test_expose_triple();
```

Comment:
No need to have so many empty lines here.
```suggestion
    test_increment_int();
    test_init_int();
    test_init_array();
    test_init_static_inner();
    test_expose_int();
    test_swap_ptr();
    test_swap_nested_ptr();
    test_swap_tuple();
    test_overwrite_dangling();
    test_expose_triple();
```